### DISCUSSION TITLE
Is there an official plan to port XCT CameraView to MAUI?

### DISCUSSION THREAD

**Discussion by StonedHackerman** [CommunityToolkit/Maui#337] [Q&A] [answered]
Does anybody know, is there an official plan to port XCT CameraView to MAUI? Either here, or in the main repo into Essentials?

**Comment by TheCodeTraveler** [comment, marked as answer]
Yes! We have an open Proposal for it here: https://github.com/CommunityToolkit/Maui/issues/106

**Comment by CliffAgius** [comment]
Closed as answered
